RasterPropMonitor/Core/IPageElement.cs

```diff
@@ -0,0 +1,7 @@
+namespace JSI
+{
+    internal interface IPageElement
```

Comment:
Why is this necessary?  There's already a lot of ways to customize page behaviors

Reply:
The `RasterPropMonitor` contains the state of which orbital patch is selected instead of `RPMVesselComputer`, otherwise all screens show the same thing, even if you only change one. Currently there's no way to pass the `RasterPropMonitor` instance to the pages themselves.

There are two main ways I considered. Either add another config variable into the `ConfigNode` which exposes it publicly and makes it hard to change later since you don't know if downstream dependencies use it, or use an internal interface. I chose the latter.

Reply:
I'm pretty sure that scansat or vesselviewer page states are not shared between monitors in the same pod.  Is there a way to use the same pattern?

Reply:
> Another example is generic switches
You might have to provide an example, I;m not sure what you mean when you refer to a "generic switch"

> Maybe this state could be stored in the JSIOrbitDisplay
We end up with a worse problem where we need to thread the state of the `JSIOrbitDisplay` everywhere it's needed. Even the text on the map display wouldn't sync up unless you passed it in.